test/server/graph/context.js

```diff
@@ -1,8 +1,8 @@
 const expect = require('chai').expect;
 
-const User = require('../../models/user');
-const Context = require('../../graph/context');
-const errors = require('../../errors');
+const User = require('../../../models/user');
```

Comment:
this might be dumb, but is there a way to not have all these horrible relative paths in require statements in node?